src/frontend/js/media-management/component/edit-sidebar.tsx

```diff
@@ -64,7 +65,7 @@ export default function EditSidebar({
   // The current directory is the last element of the directory path
   const file = directoryContent[fileIndex] as File;
   // This state is a buffer for the currently changed file
-  const [changedFile, setChangedFile] = useState<File>(file);
+  const [changedFile, setChangedFile] = useState<File | null>(file);
```

Comment:
```suggestion
  const [changedFile, setChangedFile] = useState<File>(file);
```
I don't think this can be `null` - since the edit sidebar is only rendered when a file is selected, we can be sure that there is always a file object in this buffer. Or am I missing something? Was there a specific reason you changed the line?